zulipterminal/config/keys.py

```diff
@@ -133,6 +133,11 @@ class KeyBinding(TypedDict, total=False):
         'help_text': 'Cycle through autocomplete suggestions in reverse',
         'key_category': 'msg_compose',
     }),
+    ('ADD_REACTION', {
```

Comment:
Any changes to `keys.py` should be reflected in `README.md` (in the same commit).